### PR TITLE
Add translation to several strings

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/57
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/43
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/436
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1083

### CODE REVIEW

File: BackofficeBundle/DataFixtures/MongoDB/AbstractLoadRoleData.php [hunk 1]
```diff
@@ -36,10 +35,11 @@ protected function generateRole($roleName)
      *
      * @return TranslatedValue
      */
-    protected function generateTranslatedValue($language, $value)
+    protected function generateTranslatedValue($language, $translations, $defaultValue)
```
Comment (itkg-alavieille) [inline BackofficeBundle/DataFixtures/MongoDB/AbstractLoadRoleData.php]: update comment method

File: BackofficeBundle/Resources/public/coffee/orchestraLib.coffee [hunk 2]
```diff
@@ -194,3 +194,14 @@ launchNotification = (type, message) ->
     content: message
     color: color
     timeout: 4000
+
+#SMARTADMIN RESET LOCAL STORAGE OVERRIDE
+$.root_.on 'click', '[data-action="orchestraResetWidgets"]', (e) ->
+  console.log(e.currentTarget)
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/public/coffee/orchestraLib.coffee]: why the console.log ?

Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/coffee/orchestraLib.coffee]: remove this line

**Comment by itkg-ngilain** [comment]
@nicolasThal, @itkg-alavieille : update done

**Comment by nicolasThal** [comment]
:+1:

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**